tests/unit/test_kwargs/test_reserved_kwargs_injection.py

```diff
@@ -66,19 +63,7 @@ def route_handler(state: state_typing) -> str:  # type: ignore
         assert client.app.state.called
 
 
-class QueryParams(BaseModel):
```

Comment:
This wasn't really needed. It was just used in one place where the params could be easily constructed by hand.